hack/node-mode.yaml

```diff
@@ -0,0 +1,9 @@
+# Example ConfigMap to enable dpu-host mode with OVNKubernetes
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: node-mode-config
```

Comment:
oh, I had meant to leave the configmap as "dpu-mode-config" still... or _something_ with "dpu" in the name anyway...

Reply:
OK, changed back.